### DISCUSSION TITLE
New SettingsGroup for SettingsControls

### DISCUSSION THREAD

**Discussion by ghost1372** [CommunityToolkit/Labs-Windows#482] [Ideas]
I can see a new design for SettingsGroup in the new build of Windows 11 (23511) It is very useful to have it
FYI @niels9001  
![photo_2023-07-29_23-29-03](https://github.com/CommunityToolkit/Labs-Windows/assets/9213496/8ddc377a-e6b2-41f8-88b6-0158c9834733)

**Comment by michael-hawker** [comment]
One thing that was on the backlog to investigate was to look at having helpers for general form controls as proposed in the WinUI repo here: https://github.com/microsoft/microsoft-ui-xaml/issues/82

So, I think this would fall under that same sort of structural assistance XAML that should work for general field forms and then also hopefully be re-usuable for something like this?

**Comment by ghost1372** [comment]
since settingscontrols moved to https://github.com/CommunityToolkit/Windows i closed this one and reopened a new issue
